frontend_app_logbook/src/component/Logbook.jsx

```diff
@@ -143,23 +166,24 @@ export class Logbook extends React.Component {
       const newLogbook = entry.id
         ? replaceEntryInLogbook(prevState.logbook, entry)
         : addEntryToLogbook(prevState.logbook, entry)
+      const sortedLogbook = { entries: newLogbook.entries.toSorted((a, b) => new Date(b.datetime) - new Date(a.datetime)) }
+      this.save(sortedLogbook)
```

Comment:
do not put side effects in a setState
The save() must be put outside of the setState

Reply:
done